.infra/kube/deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tusd
+  namespace: tus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: tusd
+    spec:
+      containers:
+      - image: docker.io/kiloreux/tusd:latest
+        imagePullPolicy: Always
+        args: ["-port=8080"]
```

Comment:
The current Frey setup uses more arguments (see https://github.com/tus/tusd/blob/master/.infra/Freyfile.hcl#L161). At least `-behind-proxy` and `-max-size` are needed. Also, what size does the disk have?